ApiReview/API.Polly.Core/NoDocs/Polly.Timeout/TimeoutStrategyOptions.cs

```diff
@@ -0,0 +1,17 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+namespace Polly.Timeout;
+
+public class TimeoutStrategyOptions : ResilienceStrategyOptions
+{
+    [Range(typeof(TimeSpan), "00:00:01", "1.00:00:00")]
+    public TimeSpan Timeout { get; set; }
```

Comment:
```suggestion
    public TimeSpan DefaultTimeout { get; set; }
```

Reply:
In hedging we have:

- `HedgingDelay`
- `HedgingDelayGenerator`

It's very similar i timeout too:

- `Timeout`
- `TimeoutGenerator`

I think it's ok to keep the simpler `Timeout` name, otherwise for consistency we should also use `DefaultHedgingDelay`, which imho is too wordy.